### DISCUSSION TITLE
icat kitten: Is there a way to display the certain frame of an animation?

### DISCUSSION THREAD

**Discussion by AlexNomadrg** [kovidgoyal/kitty#7462] [Q&A] [answered]
When I open a PDF file with icat, it continuously loops through its pages. Instead, I want it to open to a specific page of the PDF.

I tried using the `--loop 1` option, but this only causes it to stop on the last page, which is not the desired behavior. Is there a way to open a specific page of the document?

**Comment by kovidgoyal** [comment, marked as answer]
No, there isnt. If you want to view pdf files use a terminal pdf viewer
for it, there are some listed in:
https://sw.kovidgoyal.net/kitty/integrations/
